engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-subscriptions-modal.controller.js

```diff
@@ -30,6 +30,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkSubscription
             function getBulkSubscriptionParams() {
                 var bulkParams = hostIds;
                 bulkParams.subscriptions = SubscriptionsHelper.getSelectedSubscriptionAmounts($scope.table);
+                bulkParams.organization_id = CurrentOrganization;
```

Comment:
Once I changed our API to return the error if Simple Content Access is enabled, this created two new requirements:

1. The incoming request must have an `organization_id` param (this was already required in the API docs, but was not actually enforced in the Rails controller)
2. The API needs to check the specified organization for Simple Content Access and respond accordingly

API requests sent by Bastion from this dialog were not even including `organization_id` as a param, despite the `:required => true` in the API docs.   I added it to `bulkParams` here to avoid the issue.